frontends/web/src/routes/buy/guide.tsx

```diff
@@ -36,9 +36,13 @@ export default function BuyGuide({ name }: BuyGuideProps) {
       }} shown={true} />
       <Entry key="guide.buy.protection" entry={{
         link: {
-          text: t('buy.info.disclaimer.privacyPolicy'),
+          text: t('buy.info.disclaimer.moonpayPrivacyPolicy'),
           url: 'https://www.moonpay.com/privacy_policy',
         },
+        link2: {
+          text: t('buy.pocket.data.link'),
```

Comment:
I would expect a similar key like moonpayPrivacyPolicy, but I expect it is probably due to that this content is shared...

on the other hand iirc it is also shared for moonpay. Could you double check why this key is not `buy.info.disclaimer.pocketPrivacyPolicy`... or why the moonpay one isn't using 'buy.moonpay.data.link'  

just fyi, we shouldn't rename existing keys, so if possible it would be great to have consistent keys.

Reply:
Yes, the content is shared with the terms and conditions. Since we would need to refactor also the keys related to moonpay, but decided not to do it now, I didn't change it.